cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -121,15 +121,22 @@ public final class Controllers {
     public static final String TIMESERIES = "timeseries";
     public static final String LOCATIONS = "locations";
 
+    public static final String SPEC_ID = "spec-id";
+    public static final String LOCATION = "location";
+    public static final String SOURCE_ENTITY = "source-entity";
+    public static final String FORECAST_DATE_TIME = "forecast-date-time";
+    public static final String ISSUE_DATE_TIME = "issue-date-time";
```

Comment:
Use "forecast-date" and "issue-date" to match the other "effective-date" etc parameters

Reply:
done